CDPBatchEditor/Commands/Command/ValueSetCommand.cs

```diff
@@ -103,6 +104,126 @@ public void MoveReferenceValuesToManualValues()
                         this.sessionService.Transactions.Add(transaction);
 
                         Console.WriteLine($"Moved {parameter.UserFriendlyShortName} = {refValue[0]} ref value to manual value and changed switch to MANUAL");
+                        }
+                    else if (parameter.ParameterType is CompoundParameterType compType)
```

Comment:
Is there missing a check if the valueset switch is actually REFERENCE? (valueSet ?.ValueSwitch == ParameterSwitchKind.REFERENCE)